ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -195,21 +202,44 @@ def create_tags(self, tags):
     def create_table_entity(self, table):
         try:
             service_name = table["database"]
-            service_type = self.database_service_map.get(
-                service_name.lower(), DatabaseServiceType.Mysql.value
-            )
-
             # TODO: use metadata.get_service_or_create
             service_entity = self.get_database_service(service_name)
-            database = Database(
-                id=uuid.uuid4(),
+            database_request = CreateDatabaseRequest(
                 name="default",
-                service=EntityReference(id=service_entity.id, type=service_type),
+                description=table.get("description"),
```

Comment:
we should't use table description as database description here 🤔, description is not an required field so if we don't have it we can skip ip

Reply:
Ok.